File: myGym/myGym/SessionModel.m
//
//  SessionModel.m
//  myGym
//
//  Created by SQLI51109 on 26/11/2015.
//  Copyright © 2015 julien gimenez. All rights reserved.
//

#import "SessionModel.h"
#import "DateManager.h"

@implementation SessionModel

- (instancetype)initWithJson:(NSDictionary*)session{
    
    self = [super init];
    if (self) {
        __id           = [session objectForKey:@"_id"];
        _name          = [session objectForKey:@"name"];
        __description  = [session objectForKey:@"_description"];
        _from          = [session objectForKey:@"from"];
        _to            = [session objectForKey:@"to"];
        _day           = [DateManager formatyyyMMddFromString:[session objectForKey:@"day"]];
        _location      = [session objectForKey:@"location"];
    }
    return self;
}

@end

File: myGymSwift/myGymSwift/SessionModel.swift
//
//  SessionModel.swift
//  myGymSwift
//
//  Created by julien gimenez on 28/11/2015.
//  Copyright © 2015 julien gimenez. All rights reserved.
//

import Foundation
import RealmSwift

class SessionModel: Object {
    
    dynamic var _id = ""
    dynamic var sport_id = ""
    dynamic var from = ""
    dynamic var duration = ""
    dynamic var location = ""
    dynamic var teacher_id = ""
    dynamic var attendance = ""
    dynamic var day = NSDate()
}
